tests/e2e/config.js

```diff
@@ -4,6 +4,7 @@ exports.config = {
   // environment
   ocis: process.env.OCIS === 'true',
   assets: './tests/e2e/filesForUpload',
+  assetsTemp: './tests/e2e/filesForUpload/temp',
```

Comment:
Now I think this would be better :smile: 
```suggestion
  tempAssetsPath: './tests/e2e/filesForUpload/temp',
```